src/js/components/Util/CustomTable.tsx

```diff
@@ -0,0 +1,121 @@
+import { useEffect, useCallback, useMemo } from 'react';
+import { Table } from 'antd';
+import type { TableColumnType } from 'antd';
+import { useSearchParams } from 'react-router-dom';
+import { useTranslatedTableColumnTitles } from '@/hooks/useTranslatedTableColumnTitles';
+import { useNotify } from '@/hooks/notifications';
+import { useTranslationFn } from '@/hooks';
+
+import type { WithVisible } from '@/types/util';
+import { EXPANDED_QUERY_PARAM_KEY } from '@/constants/table';
+
+export interface CustomTableColumn<T> extends TableColumnType<T> {
+  isEmpty?: (value: any, record?: T) => boolean; // eslint-disable-line @typescript-eslint/no-explicit-any
+  isEmptyDefaultCheck?: boolean;
```

Comment:
should this be inverted into "alwaysShow" or something? alternatively, I'd like if it were merged with isEmpty to make isEmpty be `isEmpty?: "default" | ((value: any, record?: T) => boolean)` given that isEmptyDefaultCheck cannot be used if isEmpty is set, and so having them be different props doesn't make a lot of sense.